SparkPost/2web2ui
# EO-578 Add unsubscribe rate by cohort page

**Note**: No data is returned from the two new signals endpoints from staging. You'll have to point to production. Also, this UI is hidden until 4/30.

### What Changed
- Adds Unsubscribe rate by cohort page. Page functionality is mostly copied from other pages.
- (Continuation of #847 EO-577) Replaces engagement rate by cohort fake action with the new endpoint.
- Some refactoring around the cohort keys in order to easily compose together the fill and tooltip labels from the different endpoints.

### How To Test
- Log into appteam or add the `feature_signals_v2` UI option to your account.
- Visit an engagement recency details page
- Click on the Unsubscribe tab & Engagement rate tabs
- Also: Verify the Engagement Recency preview from health score or spam traps render correctly

### To Do
- [x] Use new endpoint
